### DISCUSSION TITLE
很高兴看到又一个微信的机器人框架，能否介绍一下工作原理、与其他的微信框架有何不同。

### DISCUSSION THREAD

**Discussion by bingjiw** [wechatferry/wechatferry#2] [Q&A]
比如与 wechaty, itchat, chatgpt-on-wechat, 还有各种DLL注入的方法的优缺点的进行比较。

使用时，要不要 手机扫码二维码？等

谢谢。

**Comment by KeJunMao** [comment]
基于 [wechatferry ](https://github.com/lich0821/WeChatFerry) 项目，本项目是 wcf 的客户端集成，皆在使其更易用，更多请访问文档。
